### ISSUE TITLE
read_csv Documentation Rendering Poorly

### ISSUE THREAD

**Issue by WillAyd** [pandas-dev/pandas#22057]
Looks like the `filepath_or_buffer` argument is rendering poorly in the `read_csv` documentation:

<img width="740" alt="image" src="https://user-images.githubusercontent.com/609873/43229729-a8d47712-901a-11e8-8bf4-80d307b33bc3.png">

Should be a pretty easy doc fix:

https://github.com/pandas-dev/pandas/blob/edb71fda022c6a155717e7a25679040ee0476639/pandas/io/parsers.py#L63

**Comment by gfyoung** [comment]
Seems to be a change in the rendering code (here / upstream).  That line hasn't changed in awhile.  It looks fine on the `0.22.0` docs, but looks like what you posted starting in `0.23.0`.

**Comment by WillAyd** [comment]
@gfyoung that's correct, though I'm under the impression that we may have just been getting lucky previously. I'm not sure of any other docstring that has a continuation like that so I'm under the impression that's what needs to be fixed, but lmk if you have other ideas

**Comment by gfyoung** [comment]
@WillAyd : Oh, I agree that we can reword this.  I'll put up a PR shortly.

**Comment by jorisvandenbossche** [comment]
Normally such continuation lines should work, that is the general solution in case you need a multi-line (in essence a single line due to the `\` :)) type description.

Maybe it has to do with the fact that it is a raw (`r"""`) string?

**Comment by jorisvandenbossche** [comment]
Well, looking at the blame, we actually did change that the last months: https://github.com/pandas-dev/pandas/pull/18968/files

**Comment by gfyoung** [comment]
Ah...hmm...I didn't notice that.  🙂 - would be surprised if that was the cause, but I think the actual PR is a better solution. 🙂

**Comment by jorisvandenbossche** [comment]
Why would you be surprised? The raw means that things like `\` are in the actual string, instead of interpreted (to make it a single line) 

(but I fully agree your PR is a big improvement in the explanation of the parameter! which makes this work-around unnecessary)

**Comment by gfyoung** [comment]
> Why would you be surprised? The raw means that things like \ are in the actual string, instead of interpreted (to make it a single line)

🤦‍♂️ - oh right, of course!  I completely blanked on that fact 🙂
